SimPEG/regularization/pgi.py

```diff
@@ -648,6 +725,7 @@ def __init__(
         gmm=None,
```

Comment:
I am not seeing anything different from the `SimplePGI` here? Any chance we can only keep one of the two. Just a lot of code duplication....

Reply:
In short, the only difference is for the volume weights. All the rest is duplicated. The `pgi` classes mirror what was done with `Simple` and `Tikhonov`:
- `SimplePGI` is equivalent to `Simple`
- `PGI` is equivalent to `Tikhonov`

I am all for merging it in a single class, with a flag to turn volume weights ON and OFF. The main reason was to reflect the original `Simple` and `Tikhonov` classes.

Reply:
So, if we change `Simple` and `Tikhonov`, we could make the same changes to `pgi`

Reply:
I get it for sure. It would make my life easier to refactor if that cleanup was happening ahead. But we can tackle it on the next PR if you want to do it there.

Reply:
or could I use your latest work on `Sparse` with regards to volume weights and mirror that in `pgi`? (just worrying about consistency, but if you already have an example, we can make it happen in this PR).

Reply:
You can take a look at #1038 , but basically the rule now will be that the reg always multiply in the cell volumes. Sensitivity weights we be pre-normalize. So the PGI cell_weights should have volumes - exactly how `PGIsmallness` is implemented. So ideally just keep that one.